src/@next/Banner/FixedBannerStyle.ts

```diff
@@ -37,14 +37,16 @@ export const StyledFixedBanner = styled.div`
 
   @media (max-width: ${Breakpoints.large}) {
     padding: 16px;
-    height: 100px;
+    min-height: 100px;
+    justify-content: flex-start;
+    align-items: flex-start;
 
     svg {
       margin-top: 0px;
     }
 
     &[data-nobutton='true'] {
-      height: 52px;
+      min-height: 52px;
     }
   }
 `;
```

Comment:
According to Rebecca's comment on Slack, the icon size should be 24x24 instead of 20x20 
![image](https://github.com/glints-dev/glints-aries/assets/12844836/5419c7e1-f35d-4584-8aa9-49899ce40c5d)